src/cli_plugin/install/settings.test.js

```diff
@@ -38,6 +38,9 @@ import { parseMilliseconds, parse } from './settings';
 const SECOND = 1000;
 const MINUTE = SECOND * 60;
 
+const ORIGINAL_PLATFORM = process.platform;
+const ORIGINAL_ARCHITECTURE = process.arch;
```

Comment:
This causes tests to be inconsistent depending on which platform/architecture you're running them from.

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1346